test/lint/lint-git-commit-check.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright (c) 2017-2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Linter to check that commit messages have a new line before the body
+# or no body at all
+#
+# reference: gist.github.com/agnivade/67b42d664ece2d4210c7
```

Comment:
Thanks for including this reference!  Let's keep it out of the code though, can you move it to the body of your commit message?

Reply:
fixed